workitem/workitem_repository_blackbox_test.go

```diff
@@ -241,12 +241,11 @@ func (s *workItemRepoBlackBoxTest) TestGetCountsPerIteration() {
 	spaceRepo.Create(s.ctx, &spaceInstance)
 	assert.NotEqual(s.T(), uuid.UUID{}, spaceInstance.ID)
 	// when
-	userActive := false
 	iterationRepo := iteration.NewIterationRepository(s.DB)
 	iteration1 := iteration.Iteration{
 		Name:       "Sprint 1",
 		SpaceID:    spaceInstance.ID,
-		UserActive: &userActive,
+		UserActive: false,
```

Comment:
Remove this line, since `UserActive` will automatically be `false`.

Reply:
Removed it here b92020c